client/src/components/displayHomepage.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { withRouter } from "react-router-dom";
+import {
+    Container,
+    Row,
+    Col,
+    Card,
+    Button,
+    InputGroup,
+    FormControl
+} from "react-bootstrap";
+class RenderHomepagePictures extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            img: [],
+            name: []
+        };
+    }
+    loadImages() {
+        //Get all users that current user is following.
+        axios
+            .get("http://localhost:8000/api/follow/", {
+                headers: { token: sessionStorage.getItem("jwt") }
+            })
+            .then(res => {
+                this.setState({
+                    name: res.data.following
+                });
+                return res.data.following;
+            })
+            .then(result => {
+                //Get the photo file for each user
+                for (var i = 0; i < result.length; i++) {
+                    //Define current user.
+                    let user = {
+                        username: result[i]
+                    };
+                    //Create a promise for the files request.
+                    axios
+                        .post("http://localhost:8000/searchuser/files", user, {
+                            headers: { token: sessionStorage.getItem("jwt") }
+                        })
+                        .then(res => {
+                            //Push new information to this.state.img
+                            let currentarray = [...this.state.img];
+                            currentarray.push({
+                                image: res.data[res.data.length - 1].filename,
+                                description: res.data[res.data.length - 1].description
+                            });
+                            //Set the state to the new array
+                            this.setState({
+                                img: currentarray
+                            });
+                        });
+                }
+            });
+    }
+    componentDidMount() {
+        this.loadImages();
+    }
+    render() {
+        return (
+            <Container>
+                <Row>
+                    {this.state.img.map(item => {
```

Comment:
I see you used the template from the profile pictures which is great. Good way to save time and keep consistency throughout